docs/modules/trino/partials/supported-versions.adoc

```diff
@@ -2,6 +2,6 @@
 // This is a separate file, since it is used by both the direct Trino documentation, and the overarching
 // Stackable Platform documentation.
 
-- 476
-- 470 (deprecated)
-- 451 (LTS)
+- 451 (deprecated)
+- 476 (deprecated)
+- 477 (LTS)
```

Comment:
nit: in think in other products these are sorted from most recent as the first entry.

Reply:
Oh yes, you are right, my bad. I will update this.

It might be worth to point this out in the issue templates and the individual AsciiDoc files.

Reply:
Changed in 663362c.